api/src/test/java/com/capitalone/dashboard/service/BuildServiceTest.java

```diff
@@ -55,6 +52,8 @@
     @Mock private DashboardServiceImpl dashboardService;
     @Mock
     private CollectorService collectorService;
+    @Mock
+    private ApiSettings apiSettings;
```

Comment:
@rvema  are we using this Mock ?

Reply:
Yes the when you mock the class it provides the mocked version of the same during execution and hence it was used

Reply:
added additional testcase when the lookupDashboardForBuildDataCreate is configured to return true